config/clock.rb

```diff
@@ -7,4 +7,5 @@ class Clock
   include Clockwork
 
   every(5.minutes, 'ClockworkCheck') { ClockworkCheck.perform_async }
+  every(1.day, 'SendApplicationsToProvider', at: '01:00') { SendApplicationsToProvider.perform_async }
```

Comment:
I know we are going for a persisted Redis on production, but my preference would be build some fault-tolerance into the system. How about we code this as if we don't trust that:
- clockwork will be running at exactly 01:00 hours to trigger the job (may be down)
- tasks may disappear from the Redis queue before they are picked up (e.g. sidekiq is down and Redis loses all data or we deliberately empty all queues

I'd recommend scheduling the task to run 3 times a day (for a daily task), while making sure things are processed only once (so adding some tracking to whether a task was performed or not).

Reply:
Does it make sense to run the task every hour? 

It's already idempotent, so we don't have to worry about only-once I think.

Reply:
If things work as they should then the only time when you should pick up any applications to progress is just after midnight because we round up `edit_by` to the end of the day. Once an hour would be the simplest thing to do I guess.